src/main/webapp/app/exercises/shared/example-submission/example-submission-import/example-submission-import.component.ts

```diff
@@ -43,13 +45,19 @@ export class ExampleSubmissionImportComponent implements OnInit {
         private activeModal: NgbActiveModal,
         private exampleSubmissionService: ExampleSubmissionService,
         private pagingService: ExampleSubmissionImportPagingService,
+        private artemisTranslatePipe: ArtemisTranslatePipe,
     ) {}
 
     ngOnInit(): void {
         this.content = { resultsOnPage: [], numberOfPages: 0 };
 
         this.performSearch(this.sort, 0);
         this.performSearch(this.search, 300);
+        if (this.exercise && this.exercise.type === ExerciseType.TEXT) {
+            this.submissionSizeHint = this.artemisTranslatePipe.transform('artemisApp.exampleSubmission.textSubmissionSizeHint');
```

Comment:
Using a direct translation here leads to an error: When changing the language while on the page the tooltip remains in the initial language.
![grafik](https://user-images.githubusercontent.com/26540346/142063503-c7a90710-1a15-465e-8620-b5bc03aba756.png)

Reply:
ok, I can move this condition into template